### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by stu-hu** [hoiliu-0801/MonoTAKD#7]
<img width="2471" height="420" alt="Image" src="https://github.com/user-attachments/assets/669ba999-9469-4507-88ec-3a02d7003f36" />Warnings are issued during training and loss is nan.
Could you please tell me where the problem is?

**Comment by hoiliu-0801** [comment]
We also did not encounter the NaN issues you mentioned, but thank you for sharing this information. Could it be related to division by zero, or perhaps downweighting the loss coefficient?

**Comment by stu-hu** [comment]
Thanks for your reply. I test it further and get the following debug information.During the training process, some sample depth maps are NaN. Can we communicate further via email? Thank you very much for your help! My email is huxiaoxi@snut.edu.cn.I sent you an email a while ago

<img width="1384" height="888" alt="Image" src="https://github.com/user-attachments/assets/f57c3384-fa87-4d38-bb69-74041b58fb0c" />

**Comment by hoiliu-0801** [comment]
Where did you obtain those depth maps? Do you mean that the KITTI depth values are NaN at the beginning? Otherwise, if the depth map exists, the depth values should not be NaN.

**Comment by stu-hu** [comment]
I findthe download link for the depth maps from the GETTUING_STARTED.MD file in the CaDDN project. 

<img width="1835" height="881" alt="Image" src="https://github.com/user-attachments/assets/e0b1c802-d84c-4e1d-b193-7e914a432f5d" />

Oh, by the way, the download link for the depth maps you provided is no longer valid. Can you provide a new download link?The debug information shows that the depth map for some idx is nan instead of all. For example:

<img width="599" height="94" alt="Image" src="https://github.com/user-attachments/assets/e7d99d23-f5dc-4725-8f83-6036ad06efb0" />

<img width="651" height="79" alt="Image" src="https://github.com/user-attachments/assets/2845af77-ae12-44ba-97a5-53962a4e91b8" />The training loss is not NAN at first, but becomes NAN after a period of training.

<img width="1713" height="307" alt="Image" src="https://github.com/user-attachments/assets/611dd19f-054d-4b8d-a980-9af77e02cd83" />

**Comment by hoiliu-0801** [comment]
I think you can filter out those NaN samples from both the training and testing splits.
